e2e/_suites/fleet/features/fleet_mode_agent.feature

```diff
@@ -15,6 +15,19 @@ Examples:
 | centos | tar       |
 | debian | tar       |
 
+@enroll
```

Comment:
FYI -  I wanted to keep this separate from the above 'install' step due to the underlying sub-command tested.  it would be a different failure if this broke but the other did not.